packages/lib-subject-viewers/src/VolumetricViewer/helpers/pointColor.js

```diff
@@ -3,13 +3,25 @@
 
 import { Color } from 'three'
 
-const ColorHues = [
-  205, 90, 60, 30, 0, 330, 300, 270, 240, 210, 180, 150, 120
+const SATURATION_INACTIVE = 45
+const SATURATION_ACTIVE = 100
+const LIGHTNESS_CLAMP = .75
+const COLOR_HUES = [
+  90, 60, 30, 0, 330, 300, 270, 240, 210, 180, 150, 120
 ]
-const CanvasColors = []
-const CanvasColorsInactive = []
-const ThreeColors = []
-const ThreeColorsInactive = []
+
+const colors = {
+  canvas: {
+    default: [],
+    active: [],
+    inactive: []
+  },
+  three: {
+    default: [],
+    active: [],
+    inactive: []
+  }
+}
 
 const pixelToPercent = (value) => {
```

Comment:
[minor][🤔] Wait, it's not a pixel we're converting here is it? We're converting an _RGB component value_ to percent. Though rgbComponentToPercent() or even eightBitsToPercent() isn't quite as catchy.

(This is tangential, _do not change the PR for this comment._)